### DISCUSSION TITLE
How to Import External CSS File for View Mode?

### DISCUSSION THREAD

**Discussion by Russe** [gsantner/markor#2031] [Q&A] [answered]
I know the feature "Inject -> head" to add styles for view mode of files.
I'm wondering how to import CSS rules which are stored within a separate css file? This css file is stored at the same folder with the normal Markor files.
That would be much easier to edit the css in an separate file instead of the Markor settings "Inject -> head".

**Comment by gsantner** [comment, marked as answer]
@Russe 
You can freely write html that is added to `<head>` section at time of viewing the file. Thus you should be able to use normal css external file inclusion with `<link rel="stylesheet"`

**Reply by Russe** [reply]
Thanks a lot for this hint, this works like a charm.

**Reply by amelillo** [reply]
Though it doesn't seem to work for me; I added
`<link rel="stylesheet" href="github.css">`
and the github.css file is in the same directory as the asciidoc document rendered.
Then I turn on View Mode and it's been rendered as usual.
(i.e. dark, orange h1, whatever)
I tried reverting it this way:
`<style> body.format-asciidoc {link.rel: 'stylesheet'; link.href: 'github.css';} </style>`
but still anything changes
What am I doing wrong? (because I **am** doing somenthing wrong  for sure;-) )

**Reply by Russe** [reply]
At "Inject -> head" you need to add `<link rel="stylesheet" href="github.css">`

At your css-file please add `#asciidoc_content { color: green; }` and save it.

When doing changes to the css file, you need to close and restart Markor app.
Now font color is green at asciidoc-files in the same directory like the css file.

As mentioned in another thread, best is to share your asciidoc-file as html plaintext, then you can have a look at the tags, ids and classes which are used to render the file.

I'm using an absolute path to the css file, because I have a lot of subfolders.

Hope this will help you.
